project/Dependencies.scala

```diff
@@ -128,7 +128,7 @@ object Dependencies {
   def excludeJakartaXmlBindApi = ExclusionRule("jakarta.xml.bind", "jakarta.xml.bind-api")
   def excludeJakarta(m: ModuleID): ModuleID = m.excludeAll(excludeJakartaActivationApi, excludeJakartaXmlBindApi)
 
-  val workspaceManager = excludeJakarta("bio.terra" % "workspace-manager-client" % "0.254.195-SNAPSHOT")
+  val workspaceManager = excludeJakarta("bio.terra" % "workspace-manager-client" % "0.254.208-SNAPSHOT")
```

Comment:
This upgrades the version of `workspace-manager-client`? Is that where the Leonardo ID bit comes from?

Reply:
That gets us the needed application API endpoint. The app ID comes from the wsm config.